### ISSUE TITLE
a render option for fist level and maybe an idea for sheers/fortune/silktouch

### ISSUE THREAD

**Issue by kurokuno** [Machir/IronFist#15]
maybe an overlay on your fists of wood/iron/so on almost like a glove that could be toggled via a key to show your current level it would be purely for looks and i would imagine in no way effect your balance  

as for the shear/silk/fortune thing i did have an idea on that maybe make a consumable item that gives you sheers upgrade could even render it on the gloves if you have it and as for fortune/silk touch

maybe a timed consumable like a potion ? or some other form of item that would give you a timed fortune effect ? just some ideas

**Comment by Badcholo** [comment]
yes, an enchanted look on your fist when you reach max level. Also a Fortune potion sounds awsome that would be compatible with fortune pickaxes so they don't double.

**Comment by Machir** [comment]
Seems like nice ideas, however they seem quite hard to implement too. Not sure how to overlay the fist with a different texture. Potions could work though, their effects are if I remember correctly, quite easy to work with.

**Comment by Badcholo** [comment]
Maybe using Baubles mod? A Glove or Ring that can be enchanted?
possibly to remove the bloody fist effect. I like the breaking blocks with your fist, i just don't like that it causes your fists to go bloody. It makes it seem like your a crazy person punching stone when there are perfectly useful picks around.
Mojangsters Modpack wants to use this, but instead of mining i'd like to use it for fist fights arena!!
with a slight knockback or knock out comment upon death.

**Comment by Badcholo** [comment]
potion effect on the fist sounds great to me. if possible.

**Comment by Machir** [comment]
Thanks for the Baubles idea, maybe I'll look into it. Not sure yet if I want to use any glove items and such. In a new issue, #22, I wrote down a new idea for these kind of things like enchantments and such. And well the bleeding, it was added as a drawback to using the fist. This mod was at the start meant to be silly and for fun but it seems it's going to a more eehm useful mod I guess haha. 
Oh and your level up time is configured in the mod config menu I think. Not sure if I added a multiplier for the levels or such though.
